Snabble/UI/Utilities/SwiftUI/TintedButtonStyle.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TintedButtonStyle.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 12.08.22.
+//
+//  Copyright © 2022 snabble. All rights reserved.
+//
+
+import SwiftUI
+
+/// Tinted button style using `Color.accent()` as background color and `Color.label()` as text color
+public struct AccentButtonStyle: ButtonStyle {
+    public func makeBody(configuration: Configuration) -> some View {
+        configuration.label
+            .padding(15)
+            .background(Color.accent())
+            .foregroundColor(Color.label())
+            .clipShape(RoundedRectangle(cornerRadius: 10))
+    }
+}
```

Comment:
Hast du eine Ahnung warum es in Türkis angezeigt wird? Der Button sieht falsch aus, da es nicht das Snabble Blau ist. Der Farbwert kommt jedoch korrekt mit 0,118,187 🤔

Reply:
Das funktionierte gestern morgen korrekt und irgendwann am Morgen dann nicht mehr. Seitdem kommt auch die Fehlermeldung: 
`SnabbleSampleApp[78973:9120098] [Graphics] UIColor created with component values far outside the expected range. Set a breakpoint on UIColorBreakForOutOfRangeColorComponents to debug. This message will only be logged once.`